HISTORY.rst

```diff
@@ -1,6 +1,12 @@
 History
 =======
 
+0.12.0: 2017-01-19
```

Comment:
I have updated the date. Is this file maintained manually, or is it generated automatically and I should not have touched it at all?